### ISSUE TITLE
Consider allowing # shellcheck source=  without a following source

### ISSUE THREAD

**Issue by knirch** [koalaman/shellcheck#905]
In udhcp it populates the environment with a bunch of lower case variables. I've created a file with the possible variables and put example content in them all (to not silence proper space warnings).

It would be nice if I didn't have to do

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/bash
# shellcheck source=etc/udhcp/environment_variables.txt
source /dev/null
```
